app/src/androidTest/java/com/android/feedme/test/ui/EditProfileTest.kt

```diff
@@ -146,6 +146,14 @@ class EditProfileTest {
       editPicture.assertIsDisplayed()
       editPicture.assertHasClickAction()
       editPicture.performClick()
+      // TODO inject a mock image picker
+      /*
+            profileViewModel.currentUserId = "ID_DEFAULT"
+            profileViewModel.updateProfilePicture(
+                profileViewModel,
+                Uri.parse(
+                    "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT0ouuiiXjqpSyZLcW7yXsrGl9T-FxgUQJMNSaoF3JUitDQ_t-oVrJjbCCQkfYmdGYJoeM&usqp=CAU"))
+      */
     }
```

Comment:
Why is this commented ? Can it be removed ?

Reply:
I left it in case we want to test it, it would require to mock a user with write ability which i did not have time to do